R/dataframe__frame.R

```diff
@@ -74,6 +94,15 @@
 NULL
 
 
+DataFrame_flags = method_as_property(function() {
+  out = lapply(self$columns, \(x) {
+    self[x]$to_series()$flags
```

Comment:
`to_series()` is needed here?
How about `self[, x]`?

Reply:
>  How about self[, x]?

Yes that's simpler, good catch